app/human-detector-app/config/ServerConfig.tsx

```diff
@@ -0,0 +1,4 @@
+
+export var apiLink = 'http://eyespy.tkw'
+export var loginUrlExtension = '/auth/login/'
+export var getUsersCamerasUrlExtension = '/users/%s/cameras'
```

Comment:
I'm not sure if there's a way to do string formatting using C-style format specifiers like `%s`, so you might have to hardcode manually wherever it's used:
```ts
const userId = '12321-8337y-311391-3821821';
const url = `/users/${userId}/cameras`
```
or make it a function:
```ts
export var getUsersCamerasUrlExtension = (userId: string): string => `/users/${userId}/cameras`;
```

Reply:
oops lowkey this was muscle memory from something else

Reply:
Addressed in commit 2ffca44c977e9b69bac20842194484765c5eb651